docs/source/pych/notes.rst

```diff
@@ -1,5 +1,5 @@
-Miscelanous Notes
-=================
+Miscellanous Notes
```

Comment:
Still misspelled: "Miscellaneous" is the correct spelling (note the missing 'e')

Reply:
I am deeply chagrined.